lib/ansible/modules/windows/win_firewall_profile.ps1

```diff
@@ -0,0 +1,118 @@
+#!powershell
+
+# Copyright : (c) 2019, Simon Lecoq <simon.lecoq@live.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#Requires -Module Ansible.ModuleUtils.CamelConversion
+
+# Specs and init
+$spec = @{
+    options = @{
+        profiles = @{ type = "list"; elements = "str"; choices = "Domain", "Private", "Public"; default = @( "Domain", "Private", "Public" ); aliases = @( "profile" ) }
+        enabled = @{ type = "str"; choices = "True", "False", "NotConfigured" }
```

Comment:
What does `NotConfigured` mean on a profile?

Reply:
Actually it's because the cmdlet use `GpoBoolean` (c.f. [Set-NetFirewallProfile](https://docs.microsoft.com/en-us/powershell/module/netsecurity/set-netfirewallprofile?view=win10-ps) from Microsoft docs) which is a boolean with three values : `True`, `False` and `NotConfigured` with the last option being related to GPO configuration. It should be safe to support only regular booleans as params, but it seems that a lot of default values are set as `NotConfigured` on a standard Windows so not sure whether it should be keeped or not 
```
PS C:\Users\ae> Get-NetFirewallProfile -Name 'public' | Select-Object -Property DefaultInBoundAction, DefaultOutBoundAction, LogAllowed, LogBlocked, LogIgnored

DefaultInboundAction  : NotConfigured
DefaultOutboundAction : NotConfigured
LogAllowed            : False
LogBlocked            : False
LogIgnored            : NotConfigured
```